evm/evmone/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1-benchmarking https://github.com/ewasm-benchmarking/evmone
```

Comment:
It seems v0.4.1-benchmarking is identical to v0.4.1. Shouldn't we just use upstream as-is in this case?